python/paddle/fluid/dygraph/dygraph_to_static/logical_transformer.py

```diff
@@ -34,7 +34,7 @@ def visit_UnaryOp(self, node):
         self.generic_visit(node)
         if isinstance(node.op, gast.Not):
             arg = ast_to_source_code(node.operand)
-            new_node_str = "fluid.dygraph.dygraph_to_static.convert_operators.convert_logical_not({})".format(
+            new_node_str = "paddle.jit.dygraph_to_static.convert_operators.convert_logical_not({})".format(
```

Comment:
Like the api `convert_call`, are these apis "convert_XX" directly exposed in the module `paddle.jit.dygraph_to_static` not `paddle.jit.dygraph_to_static.convert_operators` ?

Reply:
I think it is better just be same as that in the past. Even if we would like to change, it is better to be in another PR.

Reply:
LGTM, the same below.